virtual-desktop/src/assets/i18n/messages.ja.json

```diff
@@ -0,0 +1,8 @@
+{
+  "Unpin from taskbar": "アプリケーションのピン留めをはずす",
+  "Pin to taskbar": "アプリケーションをピン留めする",
+  "Bring to Front": "最前面に移動",
```

Comment:
Where did you get the Japanese translations from? Just making sure that these are correct.

Reply:
I got the translations from the dev team in Japan.